README.md

```diff
@@ -83,13 +83,17 @@ Alternatively, you can add the URI scheme in your project's Info page.
 
 ### Add your Branch Key to your project
 
-After you register your app, your app key can be retrieved on the [Settings](https://dashboard.branch.io/#/settings) page of the dashboard. Now you need to add it to YourProject-Info.plist (Info.plist for Swift).
+After you register your app, your Branch Key can be retrieved on the [Settings](https://dashboard.branch.io/#/settings) page of the dashboard. Now you need to add it to YourProject-Info.plist (Info.plist for Swift).
 
 1. In plist file, mouse hover "Information Property List" which is the root item under the Key column.
 1. After about half a second, you will see a "+" sign appear. Click it.
-1. In the newly added row, fill in "bnc_app_key" for its key, leave type as String, and enter your Branch Key obtained in above steps in its value column.
+1. In the newly added row, fill in "branch_key" for its key, leave type as String*, and enter your app's Branch Key obtained in above steps in the value column.
 1. Save the plist file.
 
+* If you want to add key for both your live and test apps at the same time, you need change this type to Dictionary, and add two entries inside:
```

Comment:
And just as a reminder, Dmitri had requested that we start pushing these into GitHub instead of S3.